src/hooks/use-dynamic-focus.hook.ts

```diff
@@ -21,10 +21,10 @@ export const useDynamicFocus = (
     }, milliSecondsWait)
   }, [ref, milliSecondsWait])
 
-  const onDynamicFocus = useCallback(async () => {
+  const onGetDynamicFocus = useCallback(async () => {
```

Comment:
since this does not `get` from the API, it should stay `onDynamicFocus`

Reply:
Handled in [3169ef1](https://github.com/ajktown/wordnote/pull/167/commits/3169ef1e95da99a8c800ed3259196109ac574f6b)